src/components/SearchBulkActions.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react'
+import { getHumanReadable } from 'helpers/getHumanReadable'
+import { Box, CheckBox, Select, Text } from 'grommet'
+import { Button } from 'components/shared/Button'
+import { Row } from 'components/shared/Row'
+
+export const SearchBulkActions = ({ results }) => {
+  const pageSizes = [10, 20, 50]
+  // the sort by option values for API and client (human-redable text)
+  const sortByOptionValues = {
+    _score: 'Best Match',
+    '-num_downloadable_samples': 'Most No. of samples',
+    num_downloadable_samples: 'Least No. of samples',
+    '-source_first_published': 'Newest Experiment',
+    source_first_published: 'Oldest Experiment'
+  }
+
+  const sortByOptions = Object.keys(sortByOptionValues).map((order) =>
+    getHumanReadable(order, sortByOptionValues)
+  )
+
+  // NOTE: when calling API, use helpers/getAPIReadable to get the API value
+  // e.g.) getAPIReadable(selectedSortByOption, sortByOptionValues)
+  const [selectedSortByOption, setSelectedSortByOption] = useState(
+    sortByOptions[0]
+  )
```

Comment:
I think this could be simplified instead of using the human readable labels as the options you can use an array of objects with attributes of `key` and `value`.

example:
https://storybook.grommet.io/?path=/story/input-select-object-options--object-options

Reply:
Updated (thank you for the link, David)!